backend/package.json

```diff
@@ -7,7 +7,7 @@
     "start": "node -r esm ./src/server.js",
     "server": "npm run start",
     "production": "cross-env NODE_ENV=production PORT=3000 node -r esm ./src/server.js",
-    "test": "jest"
+    "test": "jest --runInBand"
```

Comment:
can't run in parallel because they all try to use the same servers and the tests break. The test suite will run slower because of this but at least it will work heh